Assets/Scripts/NPC/Idle.cs

```diff
@@ -0,0 +1,31 @@
+using UnityEngine;
+
+public class Idle : SmallEnemiesBaseState
+{
+    private float _currentWaitTime;
+    [SerializeField, Range(0, 30)] private float waitTime;
+
+    protected override void EnterState(SmallEnemiesStateMachine enemy)
+    {
+        IsValidToSwitch = true;
+
+        _currentWaitTime = waitTime;
+    }
+
+    protected override void ExitState(SmallEnemiesStateMachine enemy)
+    {
+        
+    }
```

Comment:
```suggestion
    protected override void ExitState(SmallEnemiesStateMachine enemy) { }
```
Het is een functie die moet bestaan, maar niet gebruikt hier wordt. Het kan dan op 1 line gezet worden.

Reply:
Dit geldt dus voor elke lege functie met de states.